presto-delta/src/main/java/com/facebook/presto/delta/DeltaConfig.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.delta;
+
+import com.facebook.airlift.configuration.Config;
+
+import javax.validation.constraints.NotNull;
+
+public class DeltaConfig
+{
+    private boolean partitionPruningEnabled = true;
+    private boolean filterPushdownEnabled = true;
+    private boolean parquetDereferencePushdownEnabled = true;
+    private int maxSplitsBatchSize = 200;
+
+    @NotNull
+    public boolean isPartitionPruningEnabled()
```

Comment:
I put the config properties just as a back up incase there is a bug in the partition pruning code (given this is a new connector). By default it is enabled. 

Since the properties are added couple of months back, we have done more testing and found the pruning to be stable. I think it is ok to remove these properties now.